### PR TITLE
Website Redesign - Featured Single Row

### PR DESCRIPTION
**Pull request by MartinMinkov**
This PR implements component # 9 - Featured Single Row

Added lots of configuration options for the FeaturedSingleRow. While the assets/content of the component is customizable, the layouts are fixed and will behave as expected.

Desktop:
image

Tablet:
image

Mobile:
image

### CODE REVIEW

File: frontend/website-redesign/src/components/FeaturedSingleRow.re [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+
+    let description =
+      merge([Theme.Type.sectionSubhead, style([marginTop(`rem(1.))])]);
+
+    let image =
+      style([
+        position(`absolute),
+        width(`percent(100.)),
+        height(`percent(60.)),
+        maxWidth(`px(848)),
```
Comment (michellewong793) [inline frontend/website-redesign/src/components/FeaturedSingleRow.re:91]: can we convert this to rems?

File: frontend/website-redesign/src/components/FeaturedSingleRow.re [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+
+module SingleRow = {
+  module RowStyles = {
+    open Css;
+
+    let container =
+      style([
+        position(`relative),
+        width(`percent(100.)),
+        height(`px(658)),
```
Comment (michellewong793) [inline frontend/website-redesign/src/components/FeaturedSingleRow.re:37]: `rem(41.)

File: frontend/website-redesign/src/pages/Index.re [hunk 3]
```diff
@@ -58,6 +58,23 @@ let make = () => {
           },
         |]
       />
+      <FeaturedSingleRow
+        row={
+          FeaturedSingleRow.Row.rowType: ImageLeftCopyRight,
```
Comment (michellewong793) [inline frontend/website-redesign/src/pages/Index.re:63]: nice! so we can just specify which one we want in the future and create new ones

**Review by michellewong793 — APPROVED** [review]

File: frontend/website-redesign/src/pages/Index.re (hunk 3 above)
Comment (figitaki) [reply frontend/website-redesign/src/pages/Index.re:63]: Super clean.

**Pull request merged**